src/containers/Passphrase.jsx

```diff
@@ -86,7 +98,8 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
           currentPassphrase,
           newPassphrase,
           twoFactorCode,
-          twoFactorToken
+          twoFactorToken,
+          ownProps.client.getStackClient().uri
```

Comment:
nit: it may be more "future proof" if we just give the raw `client` and do the `getStackClient().uri` in related methods (but this is very subjective)

Reply:
I agree that it's subjective although there's probably an objectively best way. I intended to let the low-level functions be unaware of any cozy-client scope, and handle the URLs I give them. Thus removing complexity from the functions and adding it higher in the container component